build.gradle.kts

```diff
@@ -179,6 +181,10 @@ subprojects {
 
     maxParallelForks = Runtime.getRuntime().availableProcessors() * 2
     testLogging {
+      if (System.getenv("CI") != null) {
+          events = setOf(TestLogEvent.FAILED)
+          info.events = setOf(TestLogEvent.FAILED)
```

Comment:
Why both?

Reply:
Reverted this part, I was trying to improve github CI output while struggling with it, but its a separate problem.